main.js

```diff
@@ -336,6 +340,36 @@ document.addEventListener('DOMContentLoaded', function () {
     }
   };
 
+  //get the basic model of a specific api request
+  function getModel(callPath) {
+    let refPath =
+      swaggerJson.paths[callPath].post.requestBody.content['application/json']
```

Comment:
it may be a "post", "put", or "patch" need to determine the method in the path here

Reply:
probably should guard against unexpected missing properties. Consider [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)